libs/blocks/global-footer/global-footer.js

```diff
@@ -71,17 +71,15 @@ class Footer {
 
     if (!this.body) return;
 
-    // TODO: revisit region picker and social links decoration logic
-    const regionAnchor = this.body.querySelector('.region-selector a');
-    if (regionAnchor?.href) {
-      regionAnchor.setAttribute('href', `${regionAnchor.getAttribute('href')}#_dnt#_dnb`);
-    }
-    const socialLinks = document.querySelectorAll('.social a');
-    socialLinks.forEach((socialLink) => {
-      socialLink.setAttribute('href', `${socialLink.getAttribute('href')}#_dnb`);
-    });
+    const [region, social] = ['.region-selector', '.social'].map((selector) => this.body.querySelector(selector));
+    const [regionParent, socialParent] = [region?.parentElement, social?.parentElement];
+    [regionParent, socialParent].forEach((parent) => parent?.replaceChildren());
```

Comment:
Can we add a comment on why we remove the children and then add them again? It's not obvious when reading this code without prior knowledge

Reply:
Done 👍